src/components/input-field/examples/input-field-text.tsx

```diff
@@ -39,7 +43,8 @@ export class InputFieldTextExample {
                 value={this.value}
                 required={this.required}
                 invalid={this.invalid}
-                disabled={this.disabled}
+                disabled={this.disabled || this.readonly}
```

Comment:
The consumer shouldn't have to set both `disabled` and `readonly` to get the correct readonly styling.

Reply:
Well, I don't know how this exactly works. But probably what I did for the component itself is not enough then? 🤔 Because without `disabled={this.disabled || this.readonly}` this example doesn't make the input field both disabled and readonly in the same time. 👇

https://user-images.githubusercontent.com/35954987/118470101-aa502d00-b706-11eb-947f-4554b377f4a0.mp4


However, making other examples readonly by writing `readonly={true}` (and refreshing the page) will actually work as expected. They get both readonly and disabled in the same time (which I what we or at least I want.

Reply:
What I mean is that all the correct styles for `readonly` should be applied by setting the `readonly` attribute. The consumer shouldn't have to set the `disabled` attribute _too_.

Reply:
I can take a look at it 🙂

Reply:
I added two fixups to solve this.

Reply:
me trying to code 😁💩

so, all I wanted is that a `readonly` filed must be `disabled` as well, so that users don't tab into it or get hover effects and so forth. Also thought that this minimizes the amount of CSS I'd have to write.

Reply:
I did a quick fix for it now, so that setting `readonly` also sets `disabled` "internally". But that isn't a great long-term solution, because that also disables the link-buttons for link-fields. However, for link-fields, I think the proper solution is to show the value as a link when the field is readonly, not just show a button after the value. I think we should do that separately though. It might be that we should do that in a separate component anyway, like Jens and Johan talked about at standup. I'm not sure…

Reply:
> that also disables the link-buttons for link-fields.
Hm.... I had not thought about that. Right. 

I am not sure if having two components and switching them on the go is easier. It's of course tempting to have static components that don't have a lot of logic with them and so forth. But I can't judge whether it's worth the efforts.

> link-fields the proper solution is to show the value as a link when the field is readonly, not just show a button after the value.

Yes, it feels in those cases, that entire value should be the clickable thing.

Reply:
Check my last two commits. What do you think? It makes the trailing icons clickable, even if the field is readonly. They lose their custom colors (because I don't want to write a ton of extra CSS).

Reply:
I'm working a bit more on this @adrianschmidt. So we will soon get nice styles even for those link-buttons. Soon I'll push more fixups

Reply:
OK @adrianschmidt now I'm done. You can check the last 4 commits now, which together enable styling of link buttons for readonly fields ;)

Reply:
https://user-images.githubusercontent.com/35954987/118631661-05018b80-b7d0-11eb-816e-b044e5e875ae.mp4